src/Bundle/spec/Builder/Field/TwigFieldSpec.php

```diff
@@ -22,4 +22,14 @@ function it_creates_fields(): void
         $field->shouldHaveType(FieldInterface::class);
         $field->getName()->shouldReturn('enabled');
     }
+
+    function it_creates_fields_with_vars(): void
```

Comment:
@lchrusciel This is the reason for this PR, when using TwigField, you might want to defines some other options.

Reply:
@lchrusciel I'm wondering if an array_merge instead of replacing all options (and loose some predefined options) could be better for end users.

Reply:
`addOption` behaves more like `setOption`. What you described is something like `addOptions`

Reply:
now we have
* setOption
* setOptions
* addOptions
* removeOption